platform/lib/build/samplesBuilder.js

```diff
@@ -558,7 +572,7 @@ class SamplesBuilder {
    * @param  {Object} parsedSample
    * @return {Object}
    */
-  _getUsedComponents(parsedSample) {
+  _getUsedComponents(sample, parsedSample) {
```

Comment:
please extract this into a separate file as this might be useful in other places as well.

Reply:
ping